architecture/rhacs-cloud-service-service-description.adoc

```diff
@@ -0,0 +1,179 @@
+
+{product-title-managed-short} allows you to secure self-managed clusters that can reach out to a Central instance.
+The clusters you secure, called Secured Clusters, are managed by you, and not by Red Hat.
+Secured Cluster services include optional vulnerability scanning services, admission control services, and data collection services used for runtime monitoring and compliance.
+You install Secured Cluster services on any OpenShift or Kubernetes cluster you want to secure.
+
+[id="billing_{context}"]
+== Billing
+
+Customers can purchase a {product-title-managed-short} subscription on the Amazon Web Services (AWS) marketplace.
+The service cost is charged hourly per secured core ^[1]^.
+
+.Example
+====
+If you have established a connection to two secured clusters, each with 5 identical nodes with 8 vCPUs (such as Amazon EC2 m7g.2xlarge), the total number of secured cores is 80 (2 x 5 x 8 = 80).
+====
+
+[.small]
+--
+[1] vCPU of a node belonging to a secured cluster.
+--
+
+[id="security-and-compliance_{context}"]
+== Security and Compliance
+
+All {rh-rhacscs-first} data in the Central instance is encrypted in transit and at rest.
+The data is stored in secure storage with full replication and high availability together with regularly scheduled backups.
+The {product-title-managed-short} is available through cloud data centers that ensure optimal performance and the ability to meet data residency requirements.
+
+[id="authentication-provider_{context}"]
+=== Authentication provider
+
+When you create a Central instance using  link:https://console.redhat.com/[Red Hat Hybrid Cloud Console], authentication for the cluster administrator is configured as part of the process.
+Customers must manage all access to the Central instance as part of their integrated solution.
+For more information about the available authentication methods, see xref:../operating/manage-user-access/understanding-authentication-providers.adoc#understanding-authentication-providers[Understanding authentication providers].
+
+The default identity provider in {product-title-managed-short} is Red Hat Single Sign-On (SSO). For more information about authentication using Red Hat SSO, see xref:../installing/getting-started-rhacs-cloud-ocp.adoc#default-access-acs-console[Default access to ACS Console].
+
+
+[id="regulatory-compliance_{context}"]
+=== Regulatory compliance
+
+For the latest regulatory compliance information, see link:https://docs.openshift.com/dedicated/osd_architecture/osd_policy/policy-process-security.html[Understanding process and security for OpenShift Dedicated].
+
+[id="metrics-and-logging_{context}"]
+== Metrics and Logging
+
+[id="service-metrics_{context}"]
+=== Service metrics
+
+Service metrics are internal only.
+Red Hat provides and maintains the service at the agreed level.
```

Comment:
agreed = agreed upon? contracted? this phrasing seems odd but maybe this came from Legal.